.gitignore

```diff
@@ -1,7 +1,9 @@
-/.project
-/config.ini
-/tests/config.ini
-/tests/test_data/
-/db
-/.buildpath
-/.settings
+.project
```

Comment:
Is it intentional to remove '/' at the beginning of ignored file names? It was here to explicitly state that ignored files must be inside root directory of mmp